diff --git a/Makefile b/Makefile
index cdb86aa..57427aa 100644
--- a/Makefile
+++ b/Makefile
@@ -24,12 +24,12 @@ USAGES_ERL=$(foreach XML, $(USAGES_XML), $(call usage_xml_to_erl, $(XML)))
 ifeq ($(shell python -c 'import simplejson' 2>/dev/null && echo yes),yes)
 PYTHON=python
 else
+ifeq ($(shell python2.7 -c 'import simplejson' 2>/dev/null && echo yes),yes)
+PYTHON=python2.7
+else
 ifeq ($(shell python2.6 -c 'import simplejson' 2>/dev/null && echo yes),yes)
 PYTHON=python2.6
 else
-ifeq ($(shell python2.5 -c 'import simplejson' 2>/dev/null && echo yes),yes)
-PYTHON=python2.5
-else
 # Hmm. Missing simplejson?
 PYTHON=python
 endif
